pkg/detectors/budibase/budibase.go

```diff
@@ -0,0 +1,97 @@
+package budibase
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+type Scanner struct {
+	client *http.Client
+}
+
+// Ensure the Scanner satisfies the interface at compile time.
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	defaultClient = common.SaneHttpClient()
+	// Make sure that your group is surrounded in boundary characters such as below to reduce false positives.
+	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"budibase"}) + `\b([a-z0-9]{32}-[a-z0-9]{78})\b`)
```

Comment:
question: How were you able to determine that the group after "-" should be 78 characters long? I created a handful of test tokens and they all appeared to be 80 characters long. I was unable to get the tests to pass until I modified the regex. Do you know if there is a difference between maybe the hosted vs on-prem api tokens?

Additionally I think we can tighten the regex by limiting the alpha characters to just `a-f`, i'm pretty sure these are hex represented.

Reply:
I created the tokens from hosted budibasea and everytime it come out with 78 chars 🤔
Should I keep it as `{78,80}` to include both tokens?

Also, I'll limit the alpha characters to just `a-f` 👍

Reply:
Lets go with `{78,80}`, that should hopefully catch both types of tokens.